gems/codders-dataset/OSVDB-108582.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
This is to say I'm more in favour of just the CVE approach at least for this one.